Makefile

```diff
@@ -107,7 +107,7 @@ smoketest-no-avr:
 	tinygo build -size short -o test.elf -target=pca10040            examples/serial
 	tinygo build -size short -o test.elf -target=pca10040            examples/test
 	# test all targets/boards
-	tinygo build             -o test.elf                             examples/blinky2 # TODO: re-enable -size flag with MachO support
+	tinygo build             -o test.wasm -tags=pca10040             examples/blinky2
```

Comment:
Is this supposed to be `-tags` instead of `-target`? It's the odd-one out now.

Also, there's no longer a smoketest of no-target no-tags build (which I guess is native?)

Reply:
PS, I ask because the old smoketest used to fail on other arches, but if there's no expectation of compiling natively, then I won't continue investigating the missing support.

Reply:
Well, 0.7.0 is out now and this smoketest is still gone, but regular `go test` still fails on other arches, so I guess I still need to investigate them.

Reply:
Sorry, I think I missed this comment. I think it was `-tags` to test for board simulation, see https://play.tinygo.org/. The regular `-target=pca10040` is already tested above.